akka-stream/src/main/scala/akka/stream/javadsl/SubFlow.scala

```diff
@@ -6,6 +6,7 @@ package akka.stream.javadsl
 import akka.NotUsed
 import akka.event.LoggingAdapter
 import akka.japi.function
+import akka.japi.Pair
```

Comment:
is this import needed? the methods use `akka.japi.Pair`, which is good so that it doesn't accidentally change because of loosing this import